core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -870,6 +870,29 @@ abstract class RDD[T: ClassTag](
       preservesPartitioning)
   }
 
+  /**
+   * Return a new RDD by applying a function to each partition of this RDD, while tracking the index
+   * of the original partition.
+   *
+   * `preservesPartitioning` indicates whether the input function preserves the partitioner, which
+   * should be `false` unless this is a pair RDD and the input function doesn't modify the keys.
+   *
+   * `isOrderSensitive` indicates whether the function is order-sensitive. If it's order
```

Comment:
no big deal but I usually avoid abbreviation in the doc though.

Reply:
Ok. Fixed.